app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
 
     <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
```

Comment:
Is the `FOREGROUND_SERVICE` needed, when we declare `FOREGROUND_SERVICE_LOCATION`?

Reply:
Yes. I have jsut tested it again and without that permission the following crash occurs

`java.lang.SecurityException: Permission Denial: startForeground from pid=7011, uid=10410 requires android.permission.FOREGROUND_SERVICE`

Reply:
Okay! Thanks for check this out!